client/src/Components/RegisterPageComponents/Stepper/BodyStepper.jsx

```diff
@@ -1,22 +1,29 @@
 import React from "react";
-import { useSelector } from "react-redux";
+import { useSelector, useDispatch } from "react-redux";
 import { useFormik } from "formik";
 import * as Yup from "yup";
 import {
   StepOneComponent,
   StepThreeComponent,
   ContactInformation,
 } from "./Steps";
+import { handleNext } from "../../../Store/Slices";
 
 const SignupSchema = Yup.object().shape({
   city: Yup.string().required("حقل المدينة مطلوب"),
   phone: Yup.string()
     .length(10, "رقم الهاتف غير صحيح")
     .required("حقل رقم الهاتف مطلوب"),
   description: Yup.string().max(255, "الوصف يجب أن لا يزيد عن 255 حرف"),
+  features: Yup.array().min(1, "حقل المزايا مطلوب"),
+  gender: Yup.string().required("حقل الفئة مطلوب"),
+  monthPrice: Yup.number().moreThan(1, "قيمة الاشتراك الشهرية مطلوبة"),
+  sixMonthPrice: Yup.number().moreThan(1, "قيمة اشتراك الستة أشهر مطلوب"),
```

Comment:
remove required from textField and add your validation here
```suggestion
  sixMonthPrice: Yup.number().moreThan(1, "قيمة اشتراك الستة أشهر مطلوب").required("msg"),
,
```
the same for month price validation